the-blue-alliance-ios/Core Data/Managed.swift

```diff
@@ -1,17 +1,18 @@
 import CoreData
 
 public protocol Managed: class, NSFetchRequestResult {
-    static var entity: NSEntityDescription { get }
     static var entityName: String { get }
     var managedObjectContext: NSManagedObjectContext? { get }
 }
 
 extension Managed where Self: NSManagedObject {
     
-    public static var entity: NSEntityDescription { return entity()  }
-    
-    public static var entityName: String { return entity.name!  }
-    
+    public static var entityName: String {
+        // entity().name! seems to come back uninitialized during testing
+        // entities appear to exist inside our model - do some more digging
+        return String(NSStringFromClass(object_getClass(self)!))
```

Comment:
Let's try to fix this properly... I feel like this is related to having the tests have a host application, where our unit tests shouldn't need a host application (and probably shouldn't have a host application)